modules/configuring-node-pools-for-hcp.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * updates/updating-hosted-control-planes.adoc
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-node-pools-for-hcp_{context}"]
+= Configuring node pools for hosted control planes
+
+On hosted control planes, you can configure node pools by creating a `MachineConfig` object inside of a config map in the management cluster.
+
+//.Prerequisites
+
+//Are any prerequisites needed for this procedure? i.e., does the customer need to perform an update first?
+
+.Procedure
+
+. To create a `MachineConfig` object inside of a config map in the management cluster, enter the following information:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${CONFIGMAP_NAME}
```

Comment:
I haven't seen this formating used before in a yaml, or even throughout the docs. Might be worth setting it to like <configmap-name> or along those lines